### DISCUSSION TITLE
需要显示所有内容，不超出隐藏，显示多行

### DISCUSSION THREAD

**Discussion by Mirrors406** [hizzgdev/jsmind#573] [General]
我看到 issues 里有人提了，但是链接打不开，项目使用的是 4.6 版本，且无法升级

**Comment by hizzgdev** [comment]
对于4.6这个版本，可以用下面这个方案试试。
```
/*将 jsmind.css 里的内容修改成下面这样*/
jmnodes{position:absolute;z-index:2;background-color:rgba(0,0,0,0);min-width:420px;}
jmnode{position:absolute;cursor:default;max-width:400px;}
```

对于新版本的jsmind有更简单的办法，就是设置一下 [node_overflow](https://github.com/hizzgdev/jsmind/blob/master/docs/zh/2.options.md) 这个参数。
